shell/platform/android/io/flutter/embedding/engine/android/AndroidTouchProcessor.java

```diff
@@ -150,7 +150,9 @@ public boolean onGenericMotionEvent(MotionEvent event) {
     // Mouse hover support is not implemented for API < 18.
     boolean isPointerEvent = Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2
         && event.isFromSource(InputDevice.SOURCE_CLASS_POINTER);
-    if (!isPointerEvent || event.getActionMasked() != MotionEvent.ACTION_HOVER_MOVE) {
+    if (!(isPointerEvent &&
```

Comment:
Can we pull this conditional into another boolean variable?  I'm not clear on what this condition signifies.  I'm wondering what it means if its not a pointer event but it is a hover move.  I'm also wondering what it affirmatively means to be a pointer + hover move, or a pointer + scroll.

Reply:
done